src/runner.rs

```diff
@@ -237,8 +237,16 @@ impl<E: Environment> Runner<E> {
             .names("Expected", "Actual");
         let is_different = diff.diff().iter().any(|d| !matches!(d, DiffOp::Equal(_)));
         if is_different {
-            println!("Result unexpected, path:{:?}\n", path.as_ref());
-            diff.prettytable();
+            println!("Result unexpected, path:{:?}", path.as_ref());
```

Comment:
Actually we can just print diff
```rust
            println!(
                "Result unexpected, path:{:?}, diff:\n{}\n",
                path.as_ref(),
                diff
            );
```
- https://github.com/romankoblov/prettydiff/blob/master/src/text.rs#L418
<img width="754" alt="image" src="https://user-images.githubusercontent.com/3848910/209560266-8f4b8f39-476f-49c5-9ac8-1dc0c2fe489b.png">

Reply:
Yes, only `prettytable` will segfault. But I also think it's a bit spam when you make a general change that might affect many cases. This function will display lots of in-difference, which I think is not very friendly.

We can make this an option afterward, whether or not the table format got fixed.

Reply:
OK